plugins/dashboard/plugin.go

```diff
@@ -225,8 +225,12 @@ type nodestatus struct {
 }
 
 type tangleTime struct {
-	Synced    bool   `json:"synced"`
-	Time      int64  `json:"time"`
+	Synced bool  `json:"synced"`
+	CTT    int64 `json:"time"`
```

Comment:
maybe change the json field name to be in line with the others

Reply:
I didn't want to break anything when changing the JSON field name, because I don't know where else it is used (maybe some other repositories or by community members)

Reply:
Done